presidio-analyzer/presidio_analyzer/analyzer_engine.py

```diff
@@ -136,6 +138,8 @@ def analyze(
         ad_hoc_recognizers: Optional[List[EntityRecognizer]] = None,
         context: Optional[List[str]] = None,
         allow_list: Optional[List[str]] = None,
+        allow_list_match: Optional[str] = None,
```

Comment:
I was considering this, however, when
        allow_list_match: str = "exact",  
        regex_flags: Optional[int] = re.DOTALL | re.MULTILINE | re.IGNORECASE,  
are the default values,

it doesnt make any sense as "exact" would have no regex_flags.

In case this is not an issue in your opinion and it is understandable, I can implement these default values.

Reply:
That's a good point I haven't thought of, but I think it's a bit clearer than the alternatives

Reply:
Yes it makes it much cleaner. I think its good to it that way, wont be too confusing.